modules/core/include/opencv2/core/hal/intrin_rvv071.hpp

```diff
@@ -0,0 +1,2661 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
+//
+//  IMPORTANT: READ BEFORE DOWNLOADING, COPYING, INSTALLING OR USING.
+//
+//  By downloading, copying, installing or using the software you agree to this license.
+//  If you do not agree to this license, do not download, install,
+//  copy or use the software.
+//
+//
+//                          License Agreement
+//                For Open Source Computer Vision Library
+//
+// Copyright (C) 2000-2008, Intel Corporation, all rights reserved.
+// Copyright (C) 2009, Willow Garage Inc., all rights reserved.
+// Copyright (C) 2013, OpenCV Foundation, all rights reserved.
+// Copyright (C) 2015, Itseez Inc., all rights reserved.
+// Copyright (C) 2015, PingTouGe Semiconductor Co., Ltd., all rights reserved.
+// Third party copyrights are property of their respective owners.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+//   * Redistribution's of source code must retain the above copyright notice,
+//     this list of conditions and the following disclaimer.
+//
+//   * Redistribution's in binary form must reproduce the above copyright notice,
+//     this list of conditions and the following disclaimer in the documentation
+//     and/or other materials provided with the distribution.
+//
+//   * The name of the copyright holders may not be used to endorse or promote products
+//     derived from this software without specific prior written permission.
+//
+// This software is provided by the copyright holders and contributors "as is" and
+// any express or implied warranties, including, but not limited to, the implied
+// warranties of merchantability and fitness for a particular purpose are disclaimed.
+// In no event shall the Intel Corporation or contributors be liable for any direct,
```

Comment:
Please use short OpenCV license header with "authors" extra comments.

Reply:
> @damonyu1989, is it fine with you if the license header in intrin_rvv071.hpp is replaced with
> 
> ```
> // This file is part of OpenCV project.
> // It is subject to the license terms in the LICENSE file found in the top-level directory
> // of this distribution and at http://opencv.org/license.html
> 
> // Copyright (C) 2015, PingTouGe Semiconductor Co., Ltd., all rights reserved.
> ```
> 
> ??
I think it is ok for us.